extension/docs/WebComponentsIntegrationGuide.md

```diff
@@ -660,9 +708,13 @@ and has `amount` taken from `amountPlanned`. `interactionId` is matching the `ma
 }
 ```
 
-## Step 6: Capture payment (required for Klarna)
+</details>
+
+Notice that a transaction is added to the payment. The transaction is of type `Authorization` and has `amount` taken from `amountPlanned`. `interactionId` is matching the `makePaymentResponse`
```

Comment:
Is it always the case that we add a transaction here? What happens in case of error? Also, as mentioned previously `interactionId` is matching the pspReference from `makePaymentResponse`. See example text here: https://github.com/commercetools/commercetools-adyen-integration/pull/570/files#diff-a7be437c442c63e03daa96ed5fc5d01116c3ad4af71ab61b2af62dac295c0d8aR347